streamz/graph.py

```diff
@@ -40,6 +43,11 @@ def create_graph(node, graph, prior_node=None, pc=None):
                    fillcolor=node._graphviz_fillcolor)
     if prior_node:
         tt = hash(prior_node)
+        # If we emit on something other than all the upstreams vis it
+        if (isinstance(node, combine_latest)
+                and node.emit_on != node.upstreams
+                and prior_node in node.emit_on):
+            edge_kwargs['style'] = 'dashed'
```

Comment:
This check seems too specific.  Ideally the visualization code wouldn't special case stream types.  Instead, I wonder if you can handle this by adding more configuration options like `_graphviz_shape` or, better yet, generalize that to be some general object like

```python
class combine_latest(object):
    _graphviz_node = {'shape': ellipse', 'style': 'rounded,filled', ...}
    _graphviz_edge = {...}
```

(I haven't thought through much how that would work, but something like that might be a bit cleaner and avoid mixing styles in with the graphviz code.

Reply:
Hmm, ok. My main issue with that for this use case is that we don't really have the concept of a graph edge having properties in streamz. Most of the edge properties (eg if it is the emit on connection) are managed by the nodes themselves. Maybe I can create/edit the `_graphviz_edge` attribute at init/update_upstreams time so that the proper number of edges are dashed.

Maybe I can do 
```python
_graphviz_edge = {downstream: {'style': 'normal'} for downstream in downstreams}}
for downstream in downstreams:
    if downstream in emit_on:
        _graphviz_edge[downstream]['style'] = 'dashed'
```
then
```python
graph.add_edge(tt, t, **node._graphviz_edge[node])
```